src/app/redux/reducers/tableView.js

```diff
@@ -216,6 +218,15 @@ const setInitialVisibleColumns = (state, action) =>
       )(action)
     : state;
 
+const setInitialColumnOrdering = action => state =>
+  f.isEmpty(f.get("columnOrdering", state))
+    ? f.flow(
+        f.prop(["result", "columns"]),
+        columns => columns.map((column, idx) => ({ idx, id: column.id })),
```

Comment:
Nur zum Klugscheißen :D :
```suggestion
        mapIndexed(({ id }, idx) => ({ id, idx }),
```